src/test/resources/no-reflect-and-serialization-list

```diff
@@ -0,0 +1,29 @@
+org.kohsuke.github.GHBranchSync
```

Comment:
Is this the full list of classes that don't get reflected or serialized? Or is this just an example?

Reply:
Yes it is. When I created the aot metadata during the test those classes were mentioned not to use serialization / reflection. (Good test 👍 😄)

We have to decide if we also put them into the serialization / reflection files like explained or leave them out.

Reply:
I would expect this list to look more like https://github.com/quarkiverse/quarkus-github-api/blob/main/deployment/src/test/java/io/quarkiverse/githubapi/deployment/GitHubApiDotNamesTest.java#L28 . 

The ones listed here look more like ones that don't have coverage maybe? I would include all of these in json files.

Reply:
All right - I am going to adjust the lists accordingly.

Reply:
@bitwiseman / @gsmet just a small shout out. The packages `org.kohsuke.github.extras.authorization` and `org.kohsuke.github.extras.okhttp3` are not present anymore.

I adjusted the list according to the mentioned list here: https://github.com/hub4j/github-api/pull/1914#discussion_r1740202891